Resources/Prototypes/Language/languages.yml

```diff
@@ -31,6 +31,14 @@
   - nah
   - wah
 
+# The common galactic Sign Language!
+- type: language
+  id: SignLanguage
+  signLanguage: true
+  obfuscateSyllables: false
+  replacement:
+  - "*incomprehensible*"
```

Comment:
Put some things like these

```suggestion
  - signs something
  - makes weird hand gestures
```

Reply:
Replacement is never heart its is just a placeholder for Replacement value.

Reply:
Make it used

Reply:
Its uses .ftl files.
I cannot translate replacement.